src/IW/Core/Issue.hs

```diff
@@ -5,16 +5,33 @@ module IW.Core.Issue
        ) where
   
 import IW.Core.Id (Id (..))
-import IW.Core.Repo (Repo)
 
+import Database.PostgreSQL.Simple.Types (PGArray (..))
+import Database.PostgreSQL.Simple.FromField (FieldParser, pgArrayFieldParser)
+import Database.PostgreSQL.Simple.ToField (Action (..))
+
+
+newtype PGArray' a = PGArray' { fromPGArray' :: [a] }
+    deriving stock (Generic, Show, Eq)
+    deriving anyclass (ToJSON)
+
+instance (FromField a, Typeable a) => FromField (PGArray' a) where
+    fromField :: FieldParser (PGArray' a)
+    fromField f dat = PGArray' <$> fromPGArray <$> ((pgArrayFieldParser fromField) f dat)
+
+instance (ToField a) => ToField (PGArray' a) where
+    toField :: PGArray' a -> Action
+    toField = toField . PGArray . fromPGArray' 
 
 -- | Data type representing a GitHub issue.
 data Issue = Issue
-    { issueId     :: Id Issue
-    , issueNumber :: Int
-    , issueTitle  :: Text
-    , issueBody   :: Text
-    , issueUrl    :: Text
-    , issueRepoId :: Id Repo
+    { issueId       :: Id Issue
+    , issueNumber   :: Int
+    , issueTitle    :: Text
+    , issueBody     :: Text
+    , issueUrl      :: Text
+    , issueOwner    :: Text
+    , issueRepoName :: Text
+    , issueLabels   :: PGArray' Text
```

Comment:
I propose to just use `PGArray Text` without JSON instances. We haven't decided on the frontend framework yet, so the solution for JSON serialisation will depend on our choice. And that's why let's try to not complicate the code at the moment.

Reply:
Got it

Reply:
If I don't have a `ToJSON` for `Issue` it won't compile.
```
No instance for (ToJSON IW.Core.Issue.Issue)
        arising from a use of ‘hoistServer’
    • In the expression:
        hoistServer
          (Proxy @API) (runAppAsHandler env) (toServant issueServer)
      In an equation for ‘server’:
          server env
            = hoistServer
                (Proxy @API) (runAppAsHandler env) (toServant issueServer)
   |
17 | server env = hoistServer (Proxy @API) (runAppAsHandler env) (toServant issueServer)
   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```